app/controllers/application_controller.rb

```diff
@@ -1,3 +1,9 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  helper_method :show_user_name
+
+  def show_user_name
```

Comment:
you can use a before callback to set an `@current_user` instance variable, so it will be available everywhere in controllers and views